lib/vm-core

```diff
@@ -964,6 +965,14 @@ core::rename(){
     # rename config file
     mv "${VM_DS_PATH}/${_new}/${_old}.conf" "${VM_DS_PATH}/${_new}/${_new}.conf" >/dev/null 2>&1
     [ $? -eq 0 ] || util::err "changed guest directory but failed to rename configuration file"
+
+    # rename saved vm state file
+    _mv_failed=0
+    for _ext in .save .save.kern .save.meta; do
```

Comment:
This contradicts our previous approach with the asterisk.

Reply:
What about `find(1)`?

Reply:
Thanks, I think this will work, but it's not as simple as I expected. 

```sh
    (
        cd "${VM_DS_PATH}/${_new}" || exit 1
        find . -type f -depth 1 -name "${_old}.save*" | while read -r oldf; do
            newf="$(echo "${oldf}" | sed "s|${_old}|${_new}|" )"
            echo mv "${oldf} ${newf}" || _mv_failed=1
        done
    )
```